fast64_internal/utility.py

```diff
@@ -81,7 +81,7 @@ def getOrMakeVertexGroup(obj, groupName):
     return obj.vertex_groups.new(name=groupName)
 
 
-def unhideAllAndGetHiddenList(scene):
+def unhideAllAndGetHiddenState(scene):
```

Comment:
should I rename the function? idk if that was a suggestion or the diff (if it's the diff then it's weird I don't have that on local)

Reply:
It is intended to be an opaque type since callers aren't expected to use it, just pass it back to the other function
but I can make an alias for Any like `HiddenState = Any`